Author: Nick Black <nick.black@sprezzatech.com>
Description: Omit -L part from ncurses5{w,}-config output
 "ncurses5-config --libs" includes a "-L" part which is not needed
 since the library is installed in a directory in the default linker
 search path.  The value is also wrong if ncurses-bin is of a
 different architecture than libncurses5-dev.
 .
 Upstream included a similar patch in the 20110507 patchlevel to omit
 the -L part if $libdir is /usr/lib.  Until there is a way to obtain
 the multiarch path without running dpkg-architecture our patch is not
 upstreamable.
 .
 Based on Sven Joachim's patch for Debian, but using current ncurses
 code (2012-12-01 patchlevel) for SprezzOS 1.
Bug-Debian: http://bugs.debian.org/638281
Forwarded: not-needed
Last-Update: 2012-12-05

---

--- ncurses_5.9.20121201-SprezzOS1/misc/ncurses-config.in	2012-11-11 00:14:28.000000000 +0000
+++ ncurses_5.9.20121201-SprezzOS1/misc/ncurses-config.in	2012-12-06 06:11:41.726460628 +0000
@@ -109,12 +109,7 @@
 ENDECHO
 		;;
 	--libs)
-		if test "$libdir" = /usr/lib
-		then
-			LIBDIR=
-		else
-			LIBDIR=-L$libdir
-		fi
+		LIBDIR=
 		if test @TINFO_NAME@ = @LIB_NAME@ ; then
 		sed -e 's,^[ ]*,,' -e 's, [ ]*, ,g' -e 's,[ ]*$,,' <<-ENDECHO
 			$LIBDIR @EXTRA_LDFLAGS@ -l${THIS} @LIBS@
